subjects/devops/play-with-containers/audit/README.md

```diff
@@ -2,30 +2,29 @@
 
 ##### Check the Repo content
 
-A `README.md` file and all files used to create and delete and manage the student infrastructure must be submitted in the repo.
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
 
 ###### Are all the required files present?
 
-###### `.env` file is not exists in the repo?
+###### `.env` file do not exist in the repo?
 
 ###### Are all pushed files in the repo clean of any credentials or passwords?
 
-##### Ask the following questions to the group or student
+##### Ask the following questions to the group or student.

```

Comment:
Would it make sense to use a `:` instead?

Reply:
done